src/serde/value/borrowed/de.rs

```diff
@@ -784,10 +802,10 @@ impl<'de> VariantAccess<'de> for VariantRefDeserializer<'de> {
         V: Visitor<'de>,
     {
         match self.value {
-            Some(Value::Object(v)) => visitor.visit_map(ObjectAccess {
+            Some(Value::Object(v)) => visitor.visit_map(ObjectRefAccess {
                 i: v.iter(),
                 v: &Value::Static(StaticNode::Null),
-            }),//visit_object(v, visitor),
+            }),
             // FIXME
```

Comment:
Same. FIXME and commented out code not adding anything

Reply:
same, same :sob:

perhaps I should just fix the fixmes ... I will do that tomorrow

Reply:
👍